operate/data-generator/src/main/java/io/camunda/operate/data/AbstractDataGenerator.java

```diff
@@ -26,9 +26,7 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.context.annotation.DependsOn;
 
-@DependsOn("schemaStartup")
 public abstract class AbstractDataGenerator implements DataGenerator {
```

Comment:
💭 this thing confused me, but I think it's used by tests?

Reply:
You mean these classes for data generation in general? This is our test data that is generated on Operate startup if `dev-data` Spring profiles is activated. It's widely used in testing. I'm not sure how crucial it is to wait till schema creation 🤔